src/panels/lovelace/cards/hui-error-card.ts

```diff
@@ -43,7 +43,7 @@ export class HuiErrorCard extends LitElement implements LovelaceCard {
     return css`
       :host {
         display: block;
-        background-color: #ef5350;
+        background-color: var(--error-color);
         color: white;
```

Comment:
If we make the background themeable, the font color should also be themeable... Something like:
`color: var(--color-on-error, white)`

Reply:
```suggestion
        color: var(--error-card-text-color);
```